src/utils.ts

```diff
@@ -30,8 +30,8 @@ const localize = nls.config({messageFormat: nls.MessageFormat.file})();
  *************************************************************************************************************/
 export function errorHandling(errorDetails: any, label?: string, moreInfo?: string) {
     let httpErrCode = null;
-    const errMsg = localize("errorHandling.invalid.credentials", "Invalid Credentials. Please ensure the username and password for ") +
-        `\n${label}\n` + localize("errorHandling.invalid.credentials2"," are valid or this may lead to a lock-out.");
+    const errMsg = localize("errorHandling.invalid.credentials", "Invalid Credentials. Please ensure the username and password for {0}", label) +
+        localize("errorHandling.invalid.credentials2"," are valid or this may lead to a lock-out.");
```

Comment:
@JillieBeanSim This was already in the master branch code, but I think it's an opportunity to improve: Could we combine `errorHandling.invalid.credentials` and `errorHandling.invalid.credentials2` into a single localized string here? That way, it will be easier for translators when sentence fragments are not broken up across different variables.

Reply:
This may be related to #572

Reply:
addressed in latest commit